app/main.py

```diff
@@ -2,0 +1,22 @@
+from app.cafe import Cafe
+from app.errors import (
+    NotWearingMaskError,
+    OutdatedVaccineError,
+    NotVaccinatedError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    Cafe.count_friends = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except (OutdatedVaccineError, NotVaccinatedError):
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            Cafe.count_friends += 1
```

Comment:
The logic here seems to be incorrect. If a friend is not wearing a mask, you are incrementing `Cafe.count_friends`, but there is no return statement, which means the loop will continue. This might not be the intended behavior.